internal/scheduler/job/options.go

```diff
@@ -0,0 +1,80 @@
+package job
+
+import (
+	"time"
+)
+
+const (
+	defaultRunJobsLimit = 1
+	defaultPluginId     = "jpi_internal"
```

Comment:
At the risk of bikeshedding -- if you think about this as "the thing that is registering a job", I think `system` is a better choice -- the system itself is registering the job, as opposed to `internal` which could indicate "an internal registration" but not _what_ registered it.

Separately, if this is likely tied to a plugin, `jpi` ties it rather closely to the job system as opposed to the ID of the plugin itself. If that's not intentional, probably just `pi` is a better choice.

Reply:
Sure pi_system works, given the rest of your comment I assume I should also rename the job_plugin table to just plugin then.

Reply:
Yeah I think so